src/Factory/AnnotatedServiceAbstractFactory.php

```diff
@@ -46,12 +43,11 @@ public function canCreate(ContainerInterface $container, $requestedName)
     }
 
     /**
-     * @param ContainerInterface $container
      * @param string $requestedName
      * @param array|null $options
-     * @return mixed
+     * @return object
      */
-    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
+    public function __invoke(containerinterface $container, $requestedName, ?array $options = null)
```

Comment:
Please remove the method's docblock comment and add return type hinting (if possible).